features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayPrivacy.kt

```diff
@@ -166,11 +170,17 @@ enum class SessionReplayPrivacy {
         return mappersList
     }
 
-    private fun buildBase64Serializer(): Base64Serializer {
+    private fun buildBase64Serializer(
```

Comment:
I think I asked before but shouldn't we get rid of the Base64 serializer already ?

Reply:
In the next pr I'm going to rename this class and remove the base64 logic. We'll still need the majority of the logic there - the fallback mechanism to resolve the bitmaps, and the caches.